src/checkout-buttons/strategies/paypal-commerce/paypal-commerce-button-strategy.ts

```diff
@@ -68,6 +59,32 @@ export default class PaypalCommerceButtonStrategy implements CheckoutButtonStrat
         return Promise.resolve();
     }
 
+    private _setupPayment(cart: Cart): Promise<string> {
+        const cartId = cart.id;
+        const url = '/api/storefront/payment/paypalcommerce';
+        const body = { cartId };
+        const headers = {
+            'X-API-INTERNAL': INTERNAL_USE_ONLY,
+            'Content-Type': ContentType.Json,
+        };
+
+        return this._requestSender.post(url, { headers, body })
```

Comment:
Instead of using the request sender directly, you could follow the same approach as the rest of the strategies.

You can create a `PaypalCommerceRequestSender` that has a `setupPayment(carId)` public method.

Also, instead of using `.then` for promises, we prefer the newer `async/await` syntax

Reply:
Thank you!
I created PaypalCommerceRequestSender. It needs for my next task too
And changed all `. then`